cocos/core/game.ts

```diff
@@ -856,9 +844,6 @@ export class Game extends EventTarget {
         const win = window;
         let hiddenPropName: string;
 
-        // Ensure rAF and cAF
-        this._setAnimFrame();
-
```

Comment:
Please do not remove this, this is necessary for rAF and cAF global variable

Reply:
move this function to initConfig yet